Gruntfile.js

```diff
@@ -41,33 +41,15 @@ module.exports = function(grunt) {
         starttime: new Date(),
         pkg: packageInfo,
 
-        // lints Less
-        recess: {
+        less: {
             options: {
-                // Set compile and compress to false to lint
-                compile: false,
                 compress: false,
-                noIDs: true,
-                noJSPrefix: true,
-                noOverqualifying: false,
-                noUnderscores: true,
-                noUniversalSelectors: false,// true,
-                strictPropertyOrder: false, // true,
-                zeroUnits: false,
-                includePaths: ['src/css', 'src/css/*']
-            },
-            lint: {
-                files: [{
-                    expand: true,
-                    ext: '.css',
-                    dest: 'bin-debug/skins/',
-                    cwd: 'src/css/',
-                    src: '{,*/}*.less'
-                }]
+                dumpLineNumbers: false,
+                paths: ['src/css', 'src/css/*']
             },
             internal: {
                 options: {
-                    compile: true
+                    dumpLineNumbers: true
```

Comment:
This is not a valid option. It should be `'comments'|'mediaquery'|'all'` ([ref](https://github.com/less/less-docs/blob/master/content/usage/using-less-in-the-browser.md#dumplinenumbers))